pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_feats.lst

```diff
@@ -160,103 +162,103 @@ Extra Arcanist Exploit	STACK:YES	MULT:YES	CHOOSE:NOCHOICE	BONUS:VAR|ArcanistExpl
 # DUPLICATE?
 #Extra Hex	DESC:You have learned the secrets of a new hex.	BENEFIT:You gain one additional hex. You must meet the prerequisites for this hex. If you are a shaman, it must be a hex granted by your spirit rather than one from a wandering spirit.&nl; Special: You can take this feat multiple times. Each time you do, you gain another hex.	SOURCEPAGE:p.147
 
-#	Prerequisite: Amateur Investigator† or inspiration† class feature.
+#	Prerequisite: Amateur Investigator� or inspiration� class feature.
 Extra Inspiration	DESC:You are more able to draw upon inspiration than most.	BENEFIT:You gain three extra use per day of inspiration in your inspiration pool.&nl; Special: If you have levels in the investigator class, you can take this feat multiple times. Each time you do, you gain three extra uses of inspiration per day.	SOURCEPAGE:p.147
 
-#	Prerequisite: Investigator talent† class feature.
+#	Prerequisite: Investigator talent� class feature.
 Extra Investigator Talent	DESC:You learn a new way to use your training and inspiration.	BENEFIT:You gain one additional investigator talent. You must meet the prerequisites for this investigator talent.&nl; Special: You can take this feat multiple times. Each time you do, you gain another investigator talent.	SOURCEPAGE:p.147
 
-#	Prerequisite: Martial flexibility† class feature.
+#	Prerequisite: Martial flexibility� class feature.
 Extra Martial Flexibility	DESC:You are extremely versatile in a fight.	BENEFIT:You can use your martial flexibility ability three additional times per day.	SOURCEPAGE:p.147
 
-#	Prerequisite: Amateur Swashbuckler† or panache† class feature.
+#	Prerequisite: Amateur Swashbuckler� or panache� class feature.
 Extra Panache (Panache)	DESC:You have more panache than the ordinary swashbuckler.	BENEFIT:You gain two more panache points at the start of each day, and your maximum panache increases by two.&nl; Special: If you have levels in the swashbuckler class, you can take this feat multiple times. Each time you do, these benefits increase by two.	SOURCEPAGE:p.147
 
-#	Prerequisite: Arcane reservoir† class feature.
-Extra Reservoir	DESC:Your reservoir of arcane energy is greater than others'.	BENEFIT:You gain three more points in your arcane reservoir, and the maximum number of points in your arcane reservoir increases by that amount.&nl; Special: You can take this feat multiple times. Its effects stack.	SOURCEPAGE:p.147
+#	Prerequisite: Arcane reservoir� class feature.
+#Extra Reservoir		STACK:YES	MULT:YES	DESC:Your reservoir of arcane energy is greater than others'.	BENEFIT:You gain three more points in your arcane reservoir, and the maximum number of points in your arcane reservoir increases by that amount.&nl; Special: You can take this feat multiple times. Its effects stack.	SOURCEPAGE:p.147
```

Comment:
Same as comment above, commented out and created line in the alphabetized section at the top of file.